### PR TITLE
Separate integration/UI and unit tests

### PR DESCRIPTION
**Pull request by mstv**
* Separate integration/UI and unit tests
* Remove not existing namespace CommitDialog from tests

(cherry picked from commit b196ab8eeb65e8f688c6dda8ceb99eb7439acc2c)
(authored by RussKie)

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

File: IntegrationTests/UITests/UITests.csproj [hunk 1]
```diff
@@ -0,0 +1,100 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (RussKie) [reply IntegrationTests/UITests/UITests.csproj]: sure

[the remark this answers is not in the document]
Comment (RussKie) [reply IntegrationTests/UITests/UITests.csproj]: done

**Comment by mstv** [comment]
Thank you, @RussKie, especially also for everything else you do for the project and the team.

**Comment by mstv** [comment]
I tried to squash -- but I discarded it because MSVS2019 modifies the solution file (refer to 5bcab443cf109a395d56665e134d5da50c157362).

**Pull request merged**